src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        cinemaHallDao.add(cinemaHall);
+        return cinemaHall;
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id)
+                .orElseThrow(() -> new RuntimeException("cinemaHall not found"));

```

Comment:
```suggestion
                .orElseThrow(() -> new EntityNotFoundException("cinemaHall not found"));
```

provide informative message